newton/_src/usd/schemas.py

```diff
@@ -0,0 +1,289 @@
+# SPDX-FileCopyrightText: Copyright (c) 2025 The Newton Developers
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import ClassVar
+
+from ..geometry import MESH_MAXHULLVERT
+from ..usd.schema_resolver import PrimType, SchemaAttribute, SchemaResolver
+
+
+class SchemaResolverNewton(SchemaResolver):
+    name: ClassVar[str] = "newton"
+    mapping: ClassVar[dict[PrimType, dict[str, SchemaAttribute]]] = {
+        PrimType.SCENE: {
+            "time_step": SchemaAttribute("newton:timeStep", 0.002),
+            "max_solver_iterations": SchemaAttribute("newton:maxSolverIterations", 5),
+            "enable_gravity": SchemaAttribute("newton:enableGravity", True),
```

Comment:
What is the difference between `newton:enableGravity` vs just setting the builtin `physics:gravityMagnitude = 0`?